layouts/entry/single.html

```diff
@@ -1,20 +1,29 @@
 {{ define "header" }}
 {{ $author := index .Site.Data.authors ( .Params.author | default "" ) }}
-<div class="container">
-    <div class="mb-md blog-entry-header single" style="background-image: url('{{ .Site.BaseURL }}/{{ .Dir }}banner.png')">
+{{ $coauthor := index .Site.Data.authors ( .Params.coauthor | default "" ) }}
+<div class="mb-md blog-entry-header single" style="background-image: url('{{ .Site.BaseURL }}/{{ .Dir }}banner.png')"></div>
+<div class="has-text-centered">
+        <div>
+            <span class="title px-md py-sm">{{ .Title }}</span>
+        </div>        
+        {{ if and $author $coauthor }}
         <div>
-            <span class="title px-md py-sm has-background-dark">{{ .Title }}</span>
+            <span class="h3 px-md py-sm ">
+                Written by <a href="https://community.citra-emu.org/users/{{ $author.key }}">{{ $author.name }}</a> 
+                and <a href="https://community.citra-emu.org/users/{{ $coauthor.key }}">{{ $coauthor.name }}</a> 
+                on {{ .Date.Format $.Site.Params.fmt.Date }}
+            </span> 
         </div>
-        {{ if $author }}
-        <div class="mt-lg">
-            <span class="h3 px-md py-sm has-background-dark">
+        {{ else if $author }}
+        <div>
+            <span class="h3 px-md py-sm ">
```

Comment:
You probably want to remove this extra space
```suggestion
            <span class="h3 px-md py-sm">
```